modules/widgets/contact-info.php

```diff
@@ -123,6 +124,10 @@ function widget( $args, $instance ) {
 				}
 			}
 
+			if ( '' != $instance['email'] ) {
```

Comment:
If the email is invalid, maybe we shouldn't be displaying anything? What's the advantage of displaying an invalid email here?

This would also simplify the whole thing, I think:

```php
if ( is_email( $instance['email'] ) ) {
	printf(
		'<div class="confit-email"><a href="mailto:%s">%s</a></div>',
		esc_html( $instance['email'] )
	);
}
```